src/query/models/matcher.go

```diff
@@ -108,3 +110,47 @@ func (m Matchers) String() string {
 
 	return buffer.String()
 }
+
+// TODO: make this more robust, handle types other than MatchEqual
+func matcherFromString(s string) (Matcher, error) {
+	ss := strings.Split(s, ":")
+	l := len(ss)
+	if len(ss) > 2 {
```

Comment:
u already have a variable for that

Reply:
I just wanted to bump by characters per commit metrics